script/gettools.sh

```diff
@@ -1,16 +1,21 @@
 #!/bin/bash -e
 
-# Note: Make sure the versions match the ones in devops-infra/docker/cipipeline/Dockerfile
-
-if [ "$(protoc --version|cut -f2 -d\ )" != "3.19.4" ]; then
-    echo "Please install correct protoc version"
-	echo "expected 3.19.4"
-	echo "currently installed $(protoc --version)"
-	exit 1
-else
-	echo "You have the correct version of protoc $(protoc --version)"
+if ! command -v buf; then
+	# https://docs.buf.build/installation#binary
+	echo "buf not detected in the path, installing it!"
+	# Substitute BIN for your bin directory.
+	# Substitute VERSION for the current released version.
+	BIN="/usr/local/bin" && \
+	BUF_VERSION="1.7.0" && \
+	curl -sSL \
+		"https://github.com/bufbuild/buf/releases/download/v${BUF_VERSION}/buf-$(uname -s)-$(uname -m)" \
+		-o "${BIN}/buf" && \
+	chmod +x "${BIN}/buf"
 fi
 
+buf --version
+which buf
```

Comment:
Just looking at this, not sure this is necesary as well seeing it's in the list of things we install, see line 19